docs/changelog.rst

```diff
@@ -1,8 +1,8 @@
 Changelog
 ~~~~~~~~~
 
-0.24.0 (September 2022)
------------------------
+0.24.0 (October 2022)
+---------------------
 
 * Updated pytket version requirement to 1.7.
```

Comment:
Should change this too,

Reply:
Had the plan just to wait merging this PR until the release is out and update the pytket version. Or do you mean something else?

Reply:
OK, that's fine then.